package.json

```diff
@@ -5,15 +5,15 @@
   "author": "Firebase (https://firebase.google.com/)",
   "license": "Apache-2.0",
   "scripts": {
-    "bootstrap": "lerna bootstrap --no-ci",
-    "lint": "lerna run lint",
-    "test": "lerna run test",
-    "compile": "lerna run compile --concurrency=8"
+    "bootstrap": "pnpm recursive install",
+    "lint": "pnpm recursive run lint --workspace-concurrency=4",
+    "test": "pnpm recursive run test --workspace-concurrency=4",
+    "compile": "pnpm recursive run compile --workspace-concurrency=4"
   },
   "devDependencies": {
     "eslint": "^6.8.0",
     "eslint-plugin-promise": "^4.2.1",
-    "lerna": "^3.20.2",
```

Comment:
Does this mean Lerna isn't needed at all any more? If so, want to remove `lerna.json` as well?

Reply:
Oops yep!